cmd/timescale-prometheus/main.go

```diff
@@ -208,18 +208,44 @@ func main() {
 		ReceivedQueries:     receivedQueries,
 	}
 	writeHandler := timeHandler(httpRequestDuration, "write", api.Write(client, elector, &promMetrics))
-	http.Handle("/write", writeHandler)
-	http.Handle("/read", timeHandler(httpRequestDuration, "read", api.Read(client, &promMetrics)))
-	http.Handle("/query", queryHandler)
-	http.Handle("/query_range", queryRangeHandler)
-	http.Handle("/api/v1/query", queryHandler)
-	http.Handle("/api/v1/query_range", queryRangeHandler)
-	http.Handle("/healthz", api.Health(client))
+	router.Post("/write", writeHandler)
+
```

Comment:
I think we should document all the endpoint that we support. Doesn't have to be detailed, but a list/table of endpoints with links to the prom documentation would be good. Can be in a separate PR.